docs/reference/api/resources/images.rst

```diff
@@ -79,18 +82,20 @@ Reference
 
 .. c:type:: ImageContent
 
-    When specifying content for an :any:`Image`, you can provide:
-
-    * a string specifying an absolute or relative path to a file in a :ref:`known image
-      format <known-image-formats>`;
-    * an absolute or relative :any:`pathlib.Path` object describing a file in a
-      :ref:`known image format <known-image-formats>`;
-    * a "blob of bytes" data type (:any:`bytes`, :any:`bytearray`, or :any:`memoryview`)
-      containing raw image data in a :ref:`known image format <known-image-formats>`;
-    * an instance of :any:`toga.Image`; or
-    * if `Pillow <https://pillow.readthedocs.io/>`__ is installed, an instance of
-      :any:`PIL.Image.Image`; or
-    * an instance of the :ref:`native platform image representation <native-image-rep>`.
+    When specifying content for an :any:`Image`, you can provide any one of the following:
+
+    * A string specifying an absolute or relative path to a file in a :ref:`known image
+      format <known-image-formats>`
```

Comment:
The semicolons are intentional line ends, with `; or` on the second-to-last one.

Reply:
My bad. I don't think I'm used to seeing lists with both lists and punctuation simultaneously, so I blindly assumed it was a holdover from previous formatting. Reverted.